app/models/flood_risk_engine/enrollment_exemption.rb

```diff
@@ -20,24 +20,28 @@ def self.include_long_dredging?
       includes(:exemption).where(flood_risk_engine_exemptions: { code: Exemption::LONG_DREDGING_CODES }).any?
     end
 
-    def latest_decision
+    def accept_reject_decision_user
+      User.find_by_id(accept_reject_decision_user_id).try(:email)
+    end
+
+    def latest_comment_at_and_user_id
       comments.order(:created_at).pluck(:created_at, :user_id).last
     end
 
     def decision_at_and_user
       return [nil, nil] if comments.empty?
-      latest = latest_decision
+      latest = latest_comment_at_and_user_id
```

Comment:
comments could contain largish strings so I think the efficiency pay off, worth possible readability issues